lizard_apps/templates/lizard_apps/script.js

```diff
@@ -7,31 +7,41 @@
+         id="nine-blocks-lizard"/>
+      </svg>
+      <span class="lizard-apps-title">` + title + `</span>
+    `
+    var appsContainer = document.getElementById('lizard-apps-container');
+    appsContainer.innerHTML = `
+    <div class="lizard-apps">
+      {% for app in view.screen.applications.all %}
+        <div class="lizard-apps-icon">
+          <a href="{{ app.url }}" target="_blank" title="{{ app.name }}">
+            <img src="//{{ request.get_host }}/{{ MEDIA_URL }}{{ app.icon }}" />
```

Comment:
MEDIA_URL moet op een slash eindigen, as per docs:

https://docs.djangoproject.com/es/1.9/ref/settings/#media-url

Bij mij werkt jou code nu niet, omdat er  nu een dubbele slash in de URL staat.